model/build.gradle.kts

```diff
@@ -39,6 +39,19 @@ spine {
     enableJava()
 }
 
+configurations {
+    create("test")
+}
+
+tasks.register<Jar>("testArchive") {
```

Comment:
I am not sure why this is `...Archive`. What are we archiving?

I suggest `testOutput` or `testOutputJar`.